### DISCUSSION TITLE
Premade GameBoy Player Border (Premade .bgr Included)

### DISCUSSION THREAD

**Discussion by TheSwampCoven** [profi200/open_agb_firm#125] [Show and tell]
I just saw that this project was updated with borders and just had to make a Gameboy Player themed one! I'm genuinely ecstatic that this was implemented. This project is awesome. Now the only thing missing is being able to hold down the dpad in the menus lol.

The base image I used was by a guy named [Malixx on Retropie](https://retropie.org.uk/forum/topic/16506/game-boy-player-bezels-overlays-complete-set-for-game-boy-advance).
Here's the PNG I made if you want to tweak it.

![open_agb_firm border](https://github.com/profi200/open_agb_firm/assets/141103496/839b7175-d513-4058-9d62-5e74bf0e676a)

Took about 20 minutes in GIMP cus I was trying to get the ratios perfect lol.

Lastly, [here's](https://github.com/profi200/open_agb_firm/files/12338654/open_agb_firm.border.zip) the final .bgr file; tested and working. (Remember to rename it to border.bgr before putting it in "SD:\3ds\open_agb_firm"!)

**Comment by Scokera** [comment]
Would these borders reduce the size of the screen? How does it work in practice?

**Comment by PW5190** [comment]
Speaking of borders, I'm creating a generic GBA border similar to the borders used on Game Boy and Game Boy Color games released through the Virtual Console.

Now I have one question. How do I save my border as a `.bgr` image?
